tests/__init__.py

```diff
@@ -0,0 +1,2 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think this file shouldn't be added (it's the reason for the test failures as far as I can tell).